crates/rover-client/schema.graphql

```diff
@@ -0,0 +1,7034 @@
+"""
+Direct the client to resolve this field locally, either from the cache or local resolvers.
+"""
```

Comment:
This was downloaded from studio, and we could definitely automate the download of this and _not_ commit it, but since that doesn't exist right now, we still need this file _somewhere_. I don't think this is something we should ever have to actually look into. At worst, we should periodically download a new one (maybe using the `fetch` command ;)). At best, we should automate that download and maybe `.gitignore` it.

But I don't have any reservations for the file just existing. It should be the only schema file we ever need

Reply:
perfect!!!!